src/postmate.js

```diff
@@ -197,7 +210,9 @@ export class ChildAPI {
   }
 
   emit (name, data) {
-    log(`Child: Emitting Event "${name}"`, data)
+    if (process.env.NODE_ENV !== 'production') {
```

Comment:
Why don’t we put the process.env check inside the log function?

Reply:
I was thinking that we could have something like this:
```javascript
this.debug = info?.debug // options operator
// ...
emit (name, data) {
  if (info.debug) log(`Child: Emitting Event "${name}"`, data)
}
```
...?

Reply:
@jakiestfu @Andarist if Jacob gives a 👍 with this PR, let's do it! 

I can fast provide a fast-follow with what I was thinking and then all of us can review that.

Reply:
@jakiestfu 
that wouldnt work, it would just remove contents of the `log` function, making it effectively a noop (although with some minification setups it could still leave access to `Postmate.debug` as possibly it wouldnt know that this property access is side effect free). And this wouldnt change a thing. I can recheck it, but when I've last checked it static analysis performed by minifiers wasnt just good enough to know that **call sites** of such noop function can be removed.

By wrapping each call site in we ensure that they can be trivially removed, because those conditions will be constant and minifier will know that code cannot go into `if (false) { //...` branch. And if w wrap every call site of this `log` function minifiers will be able to remove it as well because it wont be referenced anymore.

@yowainwright 
it would actually just move `this.debug`/`Postmate.debug` condition to other places and **increase** final bundle size a little, because you'd have to check this at each call site instead of single time in the `log` itself.